Java/cp/Anagram.java

```diff
@@ -0,0 +1,57 @@
+/* This is a java code of checking whether the Strings are anagram to each outher and not...
+An anagram of a string is another string that contains the same characters, only the order of characters can be different.
+For example, “abcd” and “dabc” are an anagram of each other.
+*/
```

Comment:
Make the beginner pitch more descriptive and justify the content. You can also discuss your approach herein